src/ocean/LibVersion.d

```diff
@@ -0,0 +1,9 @@
+module ocean.LibVersion;
```

Comment:
Since this module is starting to only care about new features introduced in minor versions, I wonder if it shouldn't be named `libFeatures` instead.